config/src/main/java/org/springframework/security/config/annotation/method/configuration/ReactiveMethodSecurityConfiguration.java

```diff
@@ -28,17 +29,22 @@
 import org.springframework.security.access.method.DelegatingMethodSecurityMetadataSource;
 import org.springframework.security.access.prepost.PrePostAdviceReactiveMethodInterceptor;
 import org.springframework.security.access.prepost.PrePostAnnotationSecurityMetadataSource;
+import org.springframework.security.config.core.GrantedAuthorityDefaults;
 
 import java.util.Arrays;
 
 /**
  * @author Rob Winch
+ * @author Tadaya Tsuyukubo
  * @since 5.0
  */
 @Configuration
 class ReactiveMethodSecurityConfiguration implements ImportAware {
 	private int advisorOrder;
 
+	@Autowired(required = false)
+	private GrantedAuthorityDefaults grantedAuthorityDefaults;
```

Comment:
Please add a package private setter and move the `@Autowired(required = false)` on to the setter to align with gh-6624

Reply:
@rwinch Updated the PR to add a package private setter for Spring Fu support.